pywbemtools/pywbemcli/_cmd_connection.py

```diff
@@ -80,17 +80,25 @@ def connection_export(context):
 @click.pass_obj
 def connection_show(context, name):
     """
-    Show the current connection or a WBEM connection definition.
+    Show connection info of a WBEM connection definition.
 
-    Display the parameters of the current connection, or if the optional NAME
-    argument is specified, of the named WBEM connection definition from the
-    connections file.
+    This command displays the WBEM connection definition of a single
+    connection as follows:
 
-    Examples:
+    * A named connection from the WBEM connections file if NAME argument exists.
```

Comment:
Change "if NAME argument exists" to "if the NAME argument is specified".

Reply:
DONE